src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -464,6 +533,47 @@ def register_submission(self, submission: Submission, options: dict) -> dict | N
                 "intermediate.objects_api_zaakobject",
             )
 
+        # Connect variables with zaak eigenshappen (if a connection has been used in ZGW registration)
+        if variables_properties := options.get("variables_properties"):
+            mapped_variables_properties = get_variables_properties_from_submission(
+                submission, variables_properties
+            )
+
+            eigenshappen = execute_unless_result_exists(
+                partial(catalogi_client.list_eigenshappen, options["zaaktype"]),
+                submission,
+                "intermediate.eigenshap",
+            )
+
+            retrieved_eigenshappen = {
+                eigenshap["naam"]: eigenshap["url"] for eigenshap in eigenshappen
+            }
+
+            for key, value in mapped_variables_properties.items():
+                if key in retrieved_eigenshappen:
+                    eigenschap_url = furl(retrieved_eigenshappen[key])
+                    eigenschap_uuid = eigenschap_url.path.segments[-1]
+
+                    created_zaakeigenshap = execute_unless_result_exists(
+                        partial(
+                            zaken_client.create_zaakeigenschap,
+                            {
+                                "zaak": zaak["url"],
+                                "eigenschap": retrieved_eigenshappen[key],
+                                "waarde": value,
```

Comment:
I think we need to do some processing on the `value` based on the `eigenschap.specificatie.formaat`

Looking at the issues in VNG standard, I find [this](https://github.com/VNG-Realisatie/gemma-zaken/issues/1751#issuecomment-1198034861)

Which says that format "datum" is written as `YYYYMMDD`, and for "datumtijd" this becomes `YYYYMMDDHHmmSS`